src/layout/Navbar/index.jsx

```diff
@@ -765,3 +663,105 @@ const listStyles =
+// 				</NavLink>
+// 				<NavLink to={'#'}>
+// 					<div className="flex items-center hover:underline">
+// 						<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">My Shop</li>
+// 						<SlArrowRight size={20} className="ml-auto text-black " />
+// 					</div>
+// 				</NavLink>
+// 				<NavLink to={Approutes.grab.initial}>
+// 					<div className="flex items-center hover:underline">
+// 						<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">Grab</li>
+// 						<SlArrowRight size={20} className="ml-auto text-black " />
+// 					</div>
+// 				</NavLink>
+// 				{/* <NavLink to={'#'}>
+// 					<div className="flex items-center hover:underline">
+// 						<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">My Transactions</li>
+// 						<SlArrowRight size={20} className="ml-auto text-black " />
+// 					</div>
+// 				</NavLink> */}
+// 				<NavLink to={Approutes.profile.notifications}>
+// 					<div className="flex items-center hover:underline">
+// 						<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">Notifications</li>
+// 						<SlArrowRight size={20} className="ml-auto text-black " />
+// 					</div>
+// 				</NavLink>
+// 				<NavLink to={'#'}>
+// 					<div className="flex items-center hover:underline">
+// 						<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">Manage my Ads</li>
+// 						<SlArrowRight size={20} className="ml-auto text-black " />
+// 					</div>
+// 				</NavLink>
+// 				<NavLink to={'#'}>
+// 					<div className="flex items-center hover:underline">
+// 						<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">Manage my shop</li>
+// 						<SlArrowRight size={20} className="ml-auto text-black " />
+// 					</div>
+// 				</NavLink>
+// 				<NavLink to={Approutes.profile.saved}>
+// 					<div className="flex items-center hover:underline">
+// 						<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">My Saved Items</li>
+// 						<SlArrowRight size={20} className="ml-auto text-black " />
+// 					</div>
+// 				</NavLink>
+
+// 				<NavLink to={Approutes.profile.adverts}>
+// 					<div className="flex items-center hover:underline">
+// 						<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">My Adverts</li>
+// 						<SlArrowRight size={20} className="ml-auto text-black " />
+// 					</div>
+// 				</NavLink>
+// 				<NavLink to={Approutes.contactUs}>
+// 					<div className="flex items-center hover:underline">
+// 						<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">Help & Contact</li>
+// 						<SlArrowRight size={20} className="ml-auto text-black " />
+// 					</div>
+// 				</NavLink>
+// 				{isLogin && (
+// 					<NavLink to={Approutes.logout}>
+// 						<div className="flex items-center hover:underline">
+// 							<li className="text-lg max-sm:text-base -12 whitespace-nowrap ">Logout</li>
+// 						</div>
+// 					</NavLink>
+// 				)}
+// 			</ul>
+// 		</ul>
+// 	</div>
+// )}
```

Comment:
This comments, is it for a future feature of a testing.?

If we won't need it, you can take it away.